ClosedXML_Tests/Excel/Worksheets/XLSheetProtectionTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ClosedXML.Excel;
+using NUnit.Framework;
+using System;
+
+namespace ClosedXML_Tests.Excel.Worksheets
+{
+    [TestFixture]
+    public class XLSheetProtectionTests
+    {
+        [Test]
+        public void TestUnprotectWorksheetWithNoPassword()
+        {
+            using (var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"Misc\SHA512PasswordProtection.xlsx")))
+            using (var wb = new XLWorkbook(stream))
+            {
+                wb.Worksheet("Sheet1").Unprotect();
+
+                // Protected with SHA-512 password - not yet supported.
```

Comment:
I think this test should be split in two: the first unprotecting `Sheet1` and the second unprotecting `Sheet2`. Until SHA-512 is implemented the second one must be muted, but both must have correct assertions from the beginning (e.g. `Assert.IsFalse(ws.Protection.Protected);`).

Reply:
Ok, the split them in to, but note that the SHA-512 tests is just a temporary measure until we actually implement the algorithm.